terrain_planner_benchmark/launch/goaltype_benchmark.launch

```diff
@@ -1,15 +1,15 @@
 <launch>
-    <arg name="location" default="dischma"/>
+    <arg name="location" default="davosdorf"/>
     <arg name="runs" default="50"/>
 
-    <node pkg="tf" type="static_transform_publisher" name="world_map" args="0 0 0 0 0 0 world map 10"/>
```

Comment:
This would publish TF at 100Hz. In ROS 2, the static tf publisher is using transient local durability, so it's not necessary to republish.